prog/mutation.go

```diff
@@ -183,6 +185,52 @@ func (ctx *mutator) mutateArg() bool {
 	return true
 }
 
+// Select a call based on the complexity of the arguments.
+func chooseCall(p *Prog, r *randGen) (*Call, bool) {
+	var callPriorities []float64
+	var totalPrio, prio float64
+	var canMutate bool
```

Comment:
I think you can omit the declarations of `totalPrio`, `prio` and `canMutate` here, and use the `:=` operator when you assign values to them below.